sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_extensions.cs

```diff
@@ -549,5 +551,71 @@ public virtual Response RemoveUserFromGroup(string group, string userId, Request
                 throw;
             }
         }
+
+        /// <summary>
+        /// Add filtered connections to multiple groups.
+        /// </summary>
+        /// <param name="groups"> A list of groups which target connections will be added into. </param>
+        /// <param name="filter"> An OData filter which target connections satisfy. </param>
+        /// <param name="context"> The request context, which can override default behaviors of the client pipeline on a per-call basis. </param>
+        /// <returns>A <see cref="Response"/> if successful.</returns>
+        public virtual Response AddConnectionsToGroups(IEnumerable<string> groups, string filter, RequestContext context = null)
+        {
+            Argument.AssertNotNull(filter, nameof(filter));
+            Argument.AssertNotNull(groups, nameof(groups));
+
+            string json = System.Text.Json.JsonSerializer.Serialize(new { filter = filter, groups = groups });
+
+            return AddConnectionsToGroups(RequestContent.Create(json), context);
```

Comment:
There's an overload that does this for you.  You can rewrite it as:
```C#
RequestContent content = RequestContent.Create(new { filter = filter, groups = groups });
return AddConnectionsToGroups(content, context);
```

Reply:
Updated